src/Configuration/ConfigurationFactory.php

```diff
@@ -348,4 +358,19 @@ private function detectCiGithubActions(): bool
 
         return $ci->getCiName() === CiDetector::CI_GITHUB_ACTIONS;
     }
+
+    private static function pathToAbsolute(
+        ?string $path,
+        string $configDir,
+    ): ?string {
+        if ($path === null) {
+            return null;
+        }
+
+        if (Path::isAbsolute($path)) {
+            return $path;
+        }
+
+        return sprintf('%s/%s', $configDir, $path);
```

Comment:
Looks good to me. While it's a BC

1. we are still on 0.x
2. I think users mostly place `infection.json` at the root of the project, so I hope not many users will be affected. Anyway, we will note it in the release notes/changelog